python/README.md

```diff
@@ -1,20 +1,44 @@
-
-nGraph can be used directly thought it's [Python API][api_python] or [C++ API][api_cpp]. Alternatively it can be used through one of its frontends, e.g. [TensorFlow][frontend_tf], [MXNet][frontend_mxnet] and [ONNX][frontend_onnx].
+## nGraph Compiler stack
+
+[nGraph][ngraph_github] is an open-source graph compiler for Artificial 
+Neural Networks (ANNs). The nGraph Compiler stack provides an inherently 
+efficient graph-based compilation infrastructure designed to be compatible 
+with the many of the upcoming ASICs, like the Intel(R) Nervana(TM) Neural Network 
+Processors (Intel(R) Nervana(TM) NNPs), while also unlocking a massive performance 
+boost on any existing hardware targets in your neural network: both GPUs 
+and CPUs. Using its flexible infrastructure, you will find it becomes 
+much easier to create Deep Learning (DL) models that can adhere to the 
+"write once, run anywhere" mantra that enables your AI solutions to easily
+go from concept to production to scale.
+
+Frameworks using nGraph to execute workloads have shown [up to 45X] performance 
+boost compared to native implementations.
+
+### Using the Python API 
+
+nGraph can be used directly with the [Python API][api_python] described here, or 
+with the [C++ API][api_cpp] described in the [core documentation]. Alternatively, 
+its performance benefits can be realized through a frontend such as 
+[TensorFlow][frontend_tf], [MXNet][frontend_mxnet], and [ONNX][frontend_onnx].
+You can also create your own custom framework to integrate directly with the 
+[nGraph Ops] for highly-targeted graph execution.
 
 ## Installation
 
-nGraph is available as binary wheels you can install from PyPI. nGraph binary wheels are currently tested on Ubuntu 16.04 and require a CPU with AVX-512 instructions, if you're using a different system, you may want to [build][ngraph_building] nGraph from sources.
+nGraph is available as binary wheels you can install from PyPI. nGraph binary 
+wheels are currently tested on Ubuntu 16.04 and require a CPU with 
+Intel(R) Advanced Vector Extensions 512 (Intel(R) AVX-512) instructions. For 
+other systems, you may want to [build][ngraph_building] from sources.
 
 Installing nGraph Python API from PyPI is easy:
 
     pip install ngraph-core
 
 ## Usage example
 
-Using nGraph's Python API to construct a computation graph and execute a computation is simple. The following example shows how to create a simple `(A + B) * C` computation graph and calculate a result using 3 numpy arrays as input.
+Using nGraph's Python API to construct a computation graph and execute a computation is simple. The 
+following example shows how to create a minimal `(A + B) * C` computation graph and calculate a result 
+using 3 numpy arrays as input.
 
 ```python
```

Comment:
Did you check the code below, to verify it's up to date?

Reply:
Tested and works as expected